selenium-system-test/src/test/scala/htsstride/steps/DifferentNINOSuffixSteps.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package htsstride.steps
+
+import htsstride.browser.Browser
+import htsstride.pages.{CheckEligibilityPage, CreateAccountPage}
+import htsstride.pages.eligibility.CustomerEligiblePage
+import htsstride.utils.NINOGenerator
+
+class DifferentNINOSuffixSteps extends Steps with NINOGenerator {
+
+  val ninoForSuffixTest = generateEligibleNINO()
+  val ninoWithSuffixC = ninoForSuffixTest.take(8).toString + "C"
```

Comment:
you could make use of the methods in `NINOGenerator` to generate a NINO and store it (`see `currentNINO`)

Reply:
ok thanks, updated